### ISSUE TITLE
Which branch am I supposed to merge back from?

### ISSUE THREAD

**Issue by sarpik** [LukeSmithxyz/voidrice#399]
Hello, I'm trying to maintain [my own fork of voidrice](https://github.com/sarpik/voidrice) with the arch i3 setup.

I started once there was only one branch - `master`.
After @LukeSmithxyz made 2 separate branches - 1 for `archi3` and 1 for `voiddwm`, I started merging from `archi3`.

Now I noticed that Luke started to commit into the `master` branch again - is it supposed to be compatible with `archi3`, `voiddwm` or both (or neither) of the branches and from which should I update my fork?

I currently see that `This [master] branch is 93 commits ahead, 37 commits behind archi3.` Huh?

Thanks!

**Comment by kronikpillow** [comment]
you could have easily know that by looking at the content of the master branch, there is no i3 config in it ..., so your merging from the archi3 branch ofc

**Comment by sarpik** [comment]
Yeah, that's why I asked - I don't understand what going on with the master branch.
Thank you for the info, Kronik:)

**Comment by kronikpillow** [comment]
yeah me neither, but I guess we will find out when luke does a update :) either way, i stick to the archi3 branch :) and I'm trying to maintain my own fork as well, mixed settings from all 3 Luke's branches plus added some of my own, haven't been pushed yet thou, as I'm modifying the LARBS install script to be compatible with Artix Linux on runit :) (Arch without systemd) :D

**Comment by sarpik** [comment]
Yep, me too - archi3 ftw - dwm was just too much swag for me xD
And I find it nice that people customize this stuff for themselves - all thanks to Luke for the initial setup though:D

Oh damn, I've heard memes about linux without systemd xd
I know you've got your reasons - best of luck:P

**Comment by LukeSmithxyz** [comment]
Just for everyone's information, the current master branch (which is not actually the default branch on Github) will be the default soon once I finish some things. `voiddwm` will probably be deleted/merge since the master will basically be the same thing.

I'm in the progress of making LARBS compatible with other Arch-based distros, mainly Artix.
I've been using Artix with runit for several months.
The dotfiles will be compatible with all, and frankly I think that while I like Void Linux, the AUR and some other Arch features are better for long-term extensibility and make as many people happy as possible.

So basically the idea is to have one single repo of dotfiles which use dwm by default that can be deployed on any Arch-based distro independent of init system.
I really just need to do some testing on Artix with OpenRC before I finalize everything.

I might keep the i3 branch, but it should be viewed as legacy as I have no intention of returning to i3. I might actually commit some other old stuff to it (ranger configs and the like).

**Comment by sarpik** [comment]
Awesome, thanks for the heads-up, Luke!

I really hope you keep the i3 branch - I personally highly prefer it and I know there'll be more peeps like me who like it that way.

On another note - making this compatible with other arch-based distros is awesome and I wish you all the best reaching this goal!

**Comment by kronikpillow** [comment]
hey luke, I'v been using LARBS on Artix, works perfectly fine, except that you manually have to install and enable the -runit services after... on the other hand, dwm is great, but please o please, leave the archi3 branch as it is a very very good setup of i3, committing old setups like ranger configs etc sounds cool, you could also have someone maintain the archi3 branch to keep it up to date with the master branch in regards of environment variables etc etc ... thats what im attempting to to either way
